fabric-entity-events-v1/src/main/java/net/fabricmc/fabric/api/entity/event/v1/ServerLivingEntityEvents.java

```diff
@@ -42,6 +42,22 @@ public final class ServerLivingEntityEvents {
 		return true;
 	});
 
+	/**
+	 * An event that is called after an entity is damaged. This is fired from {@link LivingEntity#damage} after damage
+	 * is applied, or after that damage was blocked by a shield.
+	 *
+	 * <p>Damage dealt is the damage initially applied to the entity. Damage taken is the amount of damage the entity
+	 * actually took, after effects such as shields and extra freezing damage are applied. Damage taken does NOT include
```

Comment:
I wonder if it should be renamed to `baseDamageTaken` or something else to make it clear its not the final amount of damage?

PR looks good otherwise 👍

Reply:
sounds good, pushed change 👍